### PR TITLE
CustomerNumber will be optional

### PR DESCRIPTION
**Pull request by cdwinsnes**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/NGVhHXNY](https://trello.com/c/NGVhHXNY)

File: content/api/shipping-guide_2/api.raml [hunk 1]
```diff
@@ -294,7 +294,7 @@ documentation:
 
       Earlier, we have just returned the expected delivery **date** when you request our Shipping Guide API to get leadtime for BPAKKE_DOR-DOR/5000. Now, based on a machine learning model gathering scanning data from deliveries made in Norway, we are also predicting the **arrival window** - i.e. when the parcel is expected to arrive at the recipient address. If you are aldready presenting expected delivery date in your checkout, we strongly recommend that you consider increasing the value by also including the predicted arrival window.
```
Comment (uksolanki) [inline content/api/shipping-guide_2/api.raml:295]: typo 'aldready'

**Review by uksolanki — APPROVED** [review]
👍

File: content/api/shipping-guide_2/api.raml (hunk 1 above)
Comment (cdwinsnes) [reply content/api/shipping-guide_2/api.raml:295]: 👍 nice catch

**Pull request merged**